src/docs/asciidoc/index.adoc

```diff
@@ -669,18 +669,18 @@ The default uses an `{default-docker-image}` base image, however you can easily
 [source, groovy, subs="verbatim,attributes", role="multi-language-sample"]
 ----
 tasks.named("dockerfile") {
-  baseImage = "oracle/graalvm-ce:20.3.0-java11"
+  baseImage = "oracle/graalvm-ce:22.2.0-java17"
```

Comment:
I have created a separate PR for this https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/535